NimbleMedium/Sources/Data/NetworkAPI/Models/APIArticleResponse.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  APIArticleResponse.swift
+//  NimbleMedium
+//
+//  Created by Mark G on 06/09/2021.
+//
+
+import Foundation
+
+struct APIArticleResponse: Codable, Equatable {
```

Comment:
Please use `Decodable` instead
```suggestion
struct APIArticleResponse: Decodable, Equatable {
```